Intersect.Editor/Localization/Strings.cs

```diff
@@ -2042,6 +2047,10 @@ public struct EventConditional
 
             public static LocalizedString canstartquest = @"Can Start Quest";
 
+            public static LocalizedString checkequippedslotlbl = @"Slot";
```

Comment:
Name the string `EquipmentSlot` and change the default text to "Equipment Slot" please.